packages/requisitions/src/ResponseRequisition/api/hooks.ts

```diff
@@ -200,6 +207,43 @@ export const useCreateOutboundFromResponse = () => {
   });
 };
 
+export const useDeleteResponseLines = () => {
```

Comment:
As per previous comment -- this calls a mutation that doesn't actually do anything yet (waiting for back-end)